pkg/release/release.go

```diff
@@ -242,6 +279,10 @@ func (r *Release) Install(ctx context.Context, name string, version Version, val
 	return nil
 }
 
+// TODO Remove once we are done with migration to EnsureInstalled.
+//
+//	Issue https://github.com/giantswarm/giantswarm/issues/4355.
+//
```

Comment:
@MarcelMue I added TODO instead. This is the same as approach in `framework/resource` in general.